01-city-tours/src/style.scss

```diff
@@ -0,0 +1,32 @@
+.tour{
+
+    max-width: 60%;
+    margin: 0 auto;
+}
```

Comment:
When using React, we no longer have a single `SCSS` files for the whole project. There's this thing called "CSS Modules", where you write a style file per component. The major advantage is that the CSS is scoped to that file and you don't need to worry about naming clash. For example you can have a class `.container` in `card.css` and `footer.css`. they won't mix themselves.

Here's the docs about [how to use CSS Modules + SCSS with CRA](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/). You can enable it in your next project. You'll see the advantages right away!